doc/contribute/guidelines.rst

```diff
@@ -554,6 +570,31 @@ exceptions:
 * Avoid using binary literals (constants starting with ``0b``).
 * Avoid using non-ASCII symbols in code, unless it significantly improves
   clarity, avoid emojis in any case.
+* Use proper capitalization of nouns in code comments (e.g. ``UART`` and not
+  ``uart``, ``CMake`` and not ``cmake``).
+
+Beyond C code, the following coding style rules apply to other types of files:
+
+* CMake
+
+  * Indent with spaces, indentation is two spaces.
+  * No space between commands (e.g. ``if``) and the corresponding opening
+    bracket (e.g. ``(``).
+
+* Devicetree
+
+  * Indent with tabs.
+  * No empty lines before a dedenting ``};``.
+  * Single empty line separating nodes at the same hierarchy level.
```

Comment:
needs something like `Must have 1 space between item and value` e.g. `dma-channels = <2>;` not `dma-channels=<2>;`